app/queries/find_resources_with_expired_embargoes.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class FindResourcesWithExpiredEmbargoes
```

Comment:
This is probably not necessary. Was an artifact of a different development direction, but it is interesting to see how easy it is to do this query in Postgres JSONB .

Reply:
Yeah, I'd remove it if we're not using it. On the prod database it would be really slow without an index anyways.